libs/faad2/Makefile

```diff
@@ -59,6 +59,11 @@ endef
 TARGET_CFLAGS += $(FPIC)
 CONFIGURE_ARGS+= --without-xmms
 
+ifeq ($(CONFIG_BUILD_PATENTED),n)
```

Comment:
You can only test for 'y' equality. CONFIG_BUILD_PATENTED may also be undefined.
Try (for example):
```
TARGET_CFLAGS += $(if $(CONFIG_BUILD_PATENTED),, -DLC_ONLY_DECODER -UDRM_PS)
```
etc...